src/js/comments.js

```diff
@@ -24,26 +24,37 @@ class Comments {
 	 */
 	_renderComments () {
 		/*global Coral*/
-		const scriptElement = document.createElement('script');
-		scriptElement.src = 'https://ft-next-talk-spike.herokuapp.com/static/embed.js';
-		scriptElement.onload = () => Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'),
-			{
-				talk: 'https://ft-next-talk-spike.herokuapp.com',
-				events: (events) => {
-					events.onAny((name, data) => {
-						const message = new CustomEvent('talkEvent', {
-							detail: {
-								name,
-								data
-							}
-						});
-						parent.dispatchEvent(message);
-					});
-				}
-			}
-		);
-
-		this.oCommentsEl.appendChild(scriptElement);
+		fetch(`https://comments-auth.ft.com/v1/jwt/`, {
+			credentials: 'include'
+		}).then(res => res.json())
+			.then(json => json.token)
```

Comment:
This is making the assumption that the request has been successful and token exists. We need to add some error handling here for different situations. This will also tie in with the event tracking and we can fire events if authentication fails